orte/tools/orterun/orterun.1in

```diff
@@ -1213,8 +1213,13 @@ the masks 0001, 0002, 0004, and 0008.  In the second case, processes bind
 to all cores on successive sockets as indicated by the masks 000f and 00f0.
 The processes cycle through the processor sockets in a round-robin fashion
 as many times as are needed.  In the third case, the masks show us that
-2 cores have been bound per process.  In the fourth case, binding is
-turned off and no bindings are reported.
+2 cores have been bound per process.  Note that the mapping by slot
+implied that hardware threading was enabled, and the mapping proceeded
```

Comment:
Mapping by slot doesn't have anything to do with HTs - a "slot" is just an allocation that means you are allowed to run a process. It has nothing to do with hardware.

What that cmd line actually means is to start however many processes were requested, mapping them sequentially across each node before moving to the next node. Each process is to be assigned two processors (cores by default, or HTs if --use-hwthreads-as-cpus is given). Note that "bind-to core" may well get an error for including it for the case where cores are being used as PUs since the mapping algorithm includes a conflicting directive. I'd have to look at the mapping code to see how it handles that conflict.

Regardless, this text is incorrect in implying a connection between "slot" and hardware. The two concepts are unrelated.